session/bootstrap.go

```diff
@@ -209,6 +209,19 @@ const (
 		feedback blob NOT NULL,
 		index hist(table_id, is_index, hist_id)
 	);`
+
+	// CreateBindInfoTable stores the sql bind info which is used to update globalBindCache
+	CreateBindInfoTable = `CREATE TABLE IF NOT EXISTS mysql.bind_info (
+		original_sql varchar(1024) NOT NULL  ,
+      	bind_sql varchar(1024) NOT NULL ,
+      	default_db varchar(1024)  NOT NULL,
+		status int(10) NOT NULL DEFAULT 1,
+		create_time timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+		update_time timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+		charset varchar(100) NOT NULL,
+		collation varchar(100) NOT NULL,
+ 		UNIQUE KEY sql_db (original_sql, default_db)
```

Comment:
Add one index for update time? Or every time we need to scan the full table.

Reply:
The `UNIQUE KEY sql_db (original_sql, default_db)` is index.

Reply:
It is not enough. For sql `select * from mysql.bind_info where update_time > xx`, it is still a full table scan.